cdc/capture/capture.go

```diff
@@ -599,7 +599,7 @@ func (c *captureImpl) AsyncClose() {
 	c.captureMu.Lock()
 	defer c.captureMu.Unlock()
 	if c.processorManager != nil {
-		c.processorManager.AsyncClose()
+		c.processorManager.Close()
```

Comment:
May be we also need to change the `capture.AsyncClose` to  `capture.Close`.

Reply:
We will consider it, when examine `owner.AsyncStop`.